com.unity.shadergraph/Editor/Drawing/Inspector/PropertyDrawers/AbstractMaterialNodePropertyDrawer.cs

```diff
@@ -48,31 +53,11 @@ VisualElement CreateGUI(AbstractMaterialNode node, InspectableAttribute attribut
                     nodeSettings.Insert(0, help);
                 }
             }
-            EnumField precisionField = null;
-            if(node.canSetPrecision)
-            {
-                precisionField = new EnumField(node.precision);
-                var propertyRow = new PropertyRow(new Label("Precision"));
-                propertyRow.Add(precisionField, (field) =>
-                {
-                    field.RegisterValueChangedCallback(evt =>
-                    {
-                        if (evt.newValue.Equals(node.precision))
-                            return;
 
-                        m_setNodesAsDirtyCallback();
-                        node.owner.owner.RegisterCompleteObjectUndo("Change precision");
-                        node.precision = (Precision)evt.newValue;
-                        node.owner.ValidateGraph();
-                        m_updateNodeViewsCallback();
-                        node.Dirty(ModificationScope.Graph);
-                    });
-                });
-                if (node is Serialization.MultiJsonInternal.UnknownNodeType)
-                    precisionField.SetEnabled(false);
-                nodeSettings.Add(propertyRow);
-            }
-            propertyVisualElement = precisionField;
+            PropertyDrawerUtils.AddDefaultNodeProperties(nodeSettings, node, m_setNodesAsDirtyCallback, m_updateNodeViewsCallback);
+
+            propertyVisualElement = null;
```

Comment:
the propertyVisualElement is part of the signature of CreateGUI(), but is only used for drawers wrapping a single control, never for node drawers.